usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -100,6 +100,11 @@ if test "$BACKUP" = "DP" ; then
     # see https://github.com/rear/rear/pull/2549
     test $LD_LIBRARY_PATH && backup_tool_LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$DP_LD_LIBRARY_PATH || backup_tool_LD_LIBRARY_PATH=$DP_LD_LIBRARY_PATH
 fi
+if test "$BACKUP" = GALAXY11 ; then
```

Comment:
```suggestion
if test "$BACKUP" = "GALAXY11" ; then
```

Reply:
@alxgu 
why quoting a constant string that contains
only plain alphanumeric (A–Z a–z 0–9) characters?

Reply:
I can fix the other comparisons that show needless quoting...

TBH, this code path should be reworked to use ReaR's include mechanism to support the different backup tools instead of listing them there.

Reply:
I don't want to fix this now, I'd rather move these code paths into separate files as it should be.